lib/AST/GraphOperationInfo.cpp

```diff
@@ -148,3 +129,72 @@ std::string GraphOperationInfo::getStringAttr(unsigned attrIdx,
   auto attrValue = attr.value;
   return attrValue.getStringValue().str();
 }
+
+GraphOperationInfo::OperandMarker::OperandMarker(
+    StringRef MangledName, const GraphOperationInfo &info)
```

Comment:
Can this be const &?

Reply:
StringRef is a light weight value type (just a const char* + length) so passing it by value is good.